### PR TITLE
Create zeranoe.com.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/zeranoe.com.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!--
+	HTTP 526:
+		- www
```
Comment (J0WI) [inline src/chrome/content/rules/zeranoe.com.xml]: Same for `gstreamer`

Comment (koops76) [inline src/chrome/content/rules/zeranoe.com.xml]: @J0WI Done.

**Comment by J0WI** [comment]
Let's see if they fix it before the next release.

**Comment by koops76** [comment]
@J0WI At least message them.

**Pull request merged**